### ISSUE TITLE
Test cafe should ignore routines

### ISSUE THREAD

**Issue by RaphaelAlvez** [ArnoldSmith86/virtualtabletop#1671]
If done properly, a file updater for routines should only change the content of routines.

So if we ignore the changes done to the routines we can get feedback of if the file updater is indeed working.

**Comment by robartsd** [comment]
For any file updater, you should look at the testcafe diff to that what changed is actually what you expected to change. This applies to routines just as much as anything else.

**Comment by RaphaelAlvez** [comment]
> For any file updater, you should look at the testcafe diff to see that what changed is actually what you expected to change. This applies to routines just as much as anything else.

Yes but it leaves it up to the dev to check everything. if we compared the result of all tests ignoring the inside of all routines, we would have a stronger checkup of whether the file updater is doing its job.

this is an enhancement suggestion and not a bug support

in #1658, for example, I expect all tests to be wrong and all the testing is being done manually. It would be great if after I finish the file updater I got an actual test comparing the results instead of just telling me that the routines are not the same because that's already expected

**Comment by ArnoldSmith86** [comment]
I don't think this is a good idea.

**Comment by RaphaelAlvez** [comment]
> I don't think this is a good idea.

Why if the end result we want is to check if widgets have all the same properties in the end of operations?

It shouldn't matter if the operations are different. Only their results.
